apps/pl/webpack.pl.prod.js

```diff
@@ -22,6 +21,13 @@ const pl = {
     new webpack.DefinePlugin({
       BUILD_TARGET: JSON.stringify('pl'),
```

Comment:
Isn't this provided in `webpack.pl.dev.js`?

Reply:
it is there too, but this file doesn't use that one yet. there is not a "shared" pl config at the moment.